wallet/wallet/hd_wallet.go

```diff
@@ -166,8 +166,10 @@ func (w *HDWallet) ListKeyPairs() []KeyPair {
 // GenerateKeyPair generates a new key pair from a node, that is derived from
 // the wallet node.
 func (w *HDWallet) GenerateKeyPair(meta []Meta) (KeyPair, error) {
+	meta = w.addDefaultKeyName(meta)
+
 	if w.IsIsolated() {
-		return nil, ErrIsolatedWalletCantGenerateKeyPairs
+		return nil, ErrIsolatedWalletCantGenerateKeys
```

Comment:
```suggestion
		return nil, ErrIsolatedWalletCannotGenerateKeys
```
same suggestion for all other places